tchau/array/array_ex1.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <cstring>
+
+using namespace std;
+
+#define MIN 1
+#define MAX 100
+#define DEBUG 0
+
+
+/*
+* Function: findMissedNumberInIntegerArray
+*
+* Question: How do you find the missing number in a given integer array of 1 to 100? 
+*/
+
+
+int findMissedNumberInIntegerArray(int arr[], int min, int max)
+{
+	int sum = 0;
+	int tmpSum = 0;
+	for(int i=min; i<=max; i++)
+	{
+		tmpSum+=i;
+		sum+=arr[i-1];
+#if DEBUG
+		cout<<"Line: "<<__LINE__<< " Func: "<<__FUNCTION__;
+		cout<<"a["<<i-1<<"] = "<<arr[i-1]<<endl;
+#endif
+	}
+
+#if DEBUG
+	cout<<"Sum: "<<sum<<endl;
+	cout<<"tmpSum: "<<tmpSum<<endl;
+#endif
+
+	return tmpSum - sum;
+}
+
+
+
+int main(int argc, char const *argv[])
```

Comment:
simple testcases (hard coded or generated) without user interaction is preferred for this kind of UT test